Essentials/src/main/java/com/earth2me/essentials/I18n.java

```diff
@@ -207,5 +288,13 @@ public ResourceBundle newBundle(final String baseName, final Locale locale, fina
             }
             return bundle;
         }
+
+        @Override
+        public Locale getFallbackLocale(String baseName, Locale locale) {
+            if (baseName == null || locale == null) {
+                throw new NullPointerException();
+            }
+            return null;
```

Comment:
```suggestion
            if (baseName == null || locale == null) {
                throw new NullPointerException();
            }
            return null;
```
This will break fallbacks from `en_US` to `en`, no?

Reply:
no, works correctly